insights/parsers/kdump.py

```diff
@@ -205,19 +206,37 @@ def using_local_disk(self):
         """
         Is kdump configured to only use local disk?
 
-        The logic used here is the first of these conditions:
+        More than one dump targets will lead to kudmp service start failure.
+        So let's suppose that only one target is set here.
+
+        The logic used here is:
 
         * If 'raw' is given as an option, then the dump is local.
         * If 'ssh', 'net', 'nfs', or 'nfs4' is given, then the dump is NOT local.
+        * If '<fs type> <partition>' is given, then the dump is local.
         * Otherwise, the dump is local.
         """
-        # The previous version used iteration across self.data.keys(), which
-        # is of course non-repeatable because hash key ordering may change.
-        # So we reverted to logic.
-        return ('raw' in self.data) or (not (
-            'ssh' in self.data or 'net' in self.data or
-            'nfs' in self.data or 'nfs4' in self.data)
-        )
+        # Simplify the logic to check by:
```

Comment:
Though the code looks being simplified, the efficiency is different, better revert the check for `raw`.